recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "diffpy.labpdfproc" %}
-{% set version = "0.1.2" %}
+{% set version = "0.1.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/d/diffpy_labpdfproc/diffpy_labpdfproc-{{ version }}.tar.gz
-  sha256: 521dd9a24c062ca52d69ef5e571c8551dbff04a23bad55bb8a7cf5940eba9bf6
+  sha256: be987107c75262f2da195b13db2f0ae42e488721a44a1d6d93034e680d0737d9
```

Comment:
@yucongalicechen Could you please double this sha256 value by following `Obtain the SHA256 value from pypi` section under `Appendix 1. Instructions for Conda-forge release` in our GitLab release page?

Reply:
I found this from the website: 80e21b15b2d3db55270846fde610ac553c68408134a62aa0d2c9d3750a15ae5b